notebooks/models/system_random_forest.ipynb

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
// Komentarz w celach dyskusyjnych

ad 1 Zastanawiam się czemu ma służyć to połączenie CR powyżej 20 w jedną klasę
Czy chodzi tu o jedynie o dane już po przewidywaniu?
A jeśli przed:
to zmienianie wszystkich lvl 20+ na jakiś jeden? 
jaki wtedy? jakiś ustalony int (np. 21) czy zmieniamy nagle typ kolumny?

Potwory z takim CR to rzeczywiście jednostki i są przypadki gdy we wszystkich 3 bestiariuszach jest tylko jeden z danym lvl, ale czy powinniśmy się nimi teraz tak przejmować? tzn. czy to nie jest problem, który częściowo może zniknąć po dodaniu kolejnych bestiariuszy? Jeśli nie zniknie to i tak w docelowym projekcie nie będziemy przecież zwracać "CR: over 20" 

Przepraszam za masę pytań, ale obawiam się, iż nie do końca zrozumiałam o co chodzi 😅

Reply:
@j-adamczyk 
@Paulina100

Reply:
Słuszne pytanie. Generalnie przyjąłbym int 21 po prostu - to wtedy odpowiada zaokrągleniu w dół wyższych wartości. Chodzi mi i o dane treningowe, i testowe. Zakładam, że to jest tak bardzo edge case, że nie ma sie co przejmować, i takie połączenie jest ok. Generalnie w ogóle 2 edycja Pathfindera nie zakłada za bardzo rzeczy powyżej 20 poziomu, do tego musiałby wyjść podręcznik explicite dla takich postaci (na co się zdecydowanie nie zanosi), więc myślę, że to będzie ok. Moim zdaniem finalnie jak najbardziej można zwracać "CR: over 20", albo wyświetlać jakiś warning dla użytkownika obok przewidywanego CR, że wykryto ekstremalną wartość i takie predykcje mogą być niepewne.